examples/qualcomm/oss_scripts/llama/runner/io_manager.h

```diff
@@ -287,6 +308,9 @@ class SmartMaskIoMgr : public IoMgrBase {
   std::string prefill_forward_name_;
   std::string kv_forward_name_;
   const bool use_int64_token_{false};
+  // If the cache length is zero, it indicates a BERT model, which does not use
+  // position ids or KV cache inputs.
+  const bool is_bert_{false};
```

Comment:
why waste a byte storing this separately rather than a private method like so?

```
bool is_bert() const {
  return prefill_cache_len_ == 0;
}
```